tests/integration/test_write_read.py

```diff
@@ -100,9 +100,9 @@ def return_dummy_data(n_points):
         precice = fenicsxprecice.Adapter(MPI.COMM_WORLD, self.dummy_config)
         precice._participant = Participant(None, None, None, None)
         precice._read_data_name = self.fake_data_name
-        precice.initialize(right_boundary, self.scalar_V)
+        precice.initialize({"Dummy-Mesh": [right_boundary, self.scalar_V, None]})
```

Comment:
I think this shows the problem of a dict containing lists following a certain pattern: We are forced to explicitly write `None` here for "formal" reasons. It would be much nicer to just write `CouplingMesh(name="Dummy-Mesh", domain=right_boundary, function_space=self.scalar_V)`.